File: math/9.Palindrome_Number.ts
const isPalindrome = (x: number): boolean => {
    const xstr = x.toString();

    for (let i = 0, j = xstr.length - 1; i < j; i++, j--) {
        if (xstr[i] !== xstr[j]) {
            return false;
        }
    }
    return true;
};

File: leetcode/cpp/9_palindrome_number.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution
{
public:
    bool isPalindrome(int x)
    {
        string x_str = to_string(x);

        for (int i = 0, j = x_str.length() - 1; i < j; i++, j--)
        {
            if (x_str[i] != x_str[j])
            {
                return false;
            }
        }

        return true;
    }
};

int main()
{
}
// by Shihab Mahamud (github.com/p-nerd)
// Monday, April 11, 2022 | 02:12:50 AM (+06)
